### PR TITLE
mbiesiad-working-with-starters.md

### PR DESCRIPTION
**Pull request by mbiesiad**
Additionally, please take a look at issue [38](https://github.com/gatsbyjs/gatsby-pl/issues/38)

Best wishes

### CODE REVIEW

File: docs/docs/recipes/working-with-starters.md [hunk 1]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
 tableOfContentsDepth: 1
 ---
 
-[Starters](/docs/starters/) are boilerplate Gatsby sites maintained officially, or by the community.
+[Startery](/docs/starters/) są boilerplatem stron Gatsby'iego utrzymywanym oficjalnie lub przez społeczność.
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
[Startery](/docs/starters/) są boilerplatem stron Gatsby utrzymywanym oficjalnie lub przez społeczność.
```

File: docs/docs/recipes/working-with-starters.md [hunk 2]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
 
-## Using a starter
+## Używanie starteru
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
## Używanie startera
```

File: docs/docs/recipes/working-with-starters.md [hunk 3]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
+### Wymagania wstępne
 
-- The [Gatsby CLI](/docs/gatsby-cli) installed
+- Zainstalowane [Gatsby CLI](/docs/gatsby-cli)
 
-### Directions
+### Wskazówki
 
-1. Find the starter you'd like to use. (_The [Starter Library](/starters/?v=2) is a good place to look!_)
+1. Znajdź starter, który chcesz użyć. (_Biblioteka [Starter Library](/starters/?v=2) jest dobrym miejscem, aby rzucić okiem!_)
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
1. Znajdź starter, którego chcesz użyć. (_Biblioteka [Starter Library](/starters/?v=2) jest dobrym miejscem, aby rzucić okiem!_)
```

File: docs/docs/recipes/working-with-starters.md [hunk 4]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
 ```
 
-### Additional resources
+### Dodatkowe materiały
 
-- Follow a [more detailed guide](/docs/starters/) on using Gatsby starters.
-- Learn how to use the [Gatsby CLI](/docs/gatsby-cli) tool to use starters in [tutorial part one](/tutorial/part-one/#using-gatsby-starters)
-- Browse the [Starter Library](/starters/?v=2)
-- Check out Gatsby's [official default starter](https://github.com/gatsbyjs/gatsby-starter-default)
+- Obserwuj [bardziej szczegółowy przewodnik](/docs/starters/) dotyczący używania starterów Gatsby'iego.
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
- Sprawdź [bardziej szczegółowy przewodnik](/docs/starters/) dotyczący używania starterów Gatsby.
```

File: docs/docs/recipes/working-with-starters.md [hunk 5]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
+- Naucz się jak używać narzędzia [Gatsby CLI](/docs/gatsby-cli) do korzystania ze starterów w [tutorial part one](/tutorial/part-one/#using-gatsby-starters)
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
- Naucz się jak używać narzędzia [Gatsby CLI](/docs/gatsby-cli) do korzystania ze starterów w [pierwszej części samouczka](/tutorial/part-one/#using-gatsby-starters)
```

File: docs/docs/recipes/working-with-starters.md [hunk 6]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
+- Przeglądaj bibliotekę [Starter Library](/starters/?v=2)
+- Sprawdź [oficjalny domyślny starter](https://github.com/gatsbyjs/gatsby-starter-default) Gatsby'iego
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
- Sprawdź [oficjalny domyślny starter](https://github.com/gatsbyjs/gatsby-starter-default) Gatsby
```

File: docs/docs/recipes/working-with-starters.md [hunk 7]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
-### Additional resources
+### Dodatkowe materiały
 
-- Follow a [more detailed guide](/docs/starters/) on using Gatsby starters.
-- Learn how to use the [Gatsby CLI](/docs/gatsby-cli) tool to use starters in [tutorial part one](/tutorial/part-one/#using-gatsby-starters)
-- Browse the [Starter Library](/starters/?v=2)
-- Check out Gatsby's [official default starter](https://github.com/gatsbyjs/gatsby-starter-default)
+- Obserwuj [bardziej szczegółowy przewodnik](/docs/starters/) dotyczący używania starterów Gatsby'iego.
+- Naucz się jak używać narzędzia [Gatsby CLI](/docs/gatsby-cli) do korzystania ze starterów w [tutorial part one](/tutorial/part-one/#using-gatsby-starters)
+- Przeglądaj bibliotekę [Starter Library](/starters/?v=2)
```
Comment (maciekgrzybek) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
- Przeglądaj  [bibliotekę starterów](/starters/?v=2)
```

**Review by maciekgrzybek — CHANGES_REQUESTED** [review]

File: docs/docs/recipes/working-with-starters.md (hunk 7 above)
Comment (mbiesiad) [reply docs/docs/recipes/working-with-starters.md]: Fixed, done. Thanks a lot! Great improvements! 🔥

Comment (maciekgrzybek) [reply docs/docs/recipes/working-with-starters.md]: awesome ;)

**Review by maciekgrzybek — APPROVED** [review]

**Pull request merged**